WA.Pizza.Infrastructure/Data/Configurations/BasketEntityConfiguration.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WA.Pizza.Infrastructure.Data.Models;
+
+namespace WA.Pizza.Infrastructure.Data.Configurations
+{
+    internal class BasketEntityConfiguration : IEntityTypeConfiguration<Basket>
+    {
+        public void Configure(EntityTypeBuilder<Basket> entity)
+        {
+            // Maybe unnecessary because has PK next line.
+            entity
+                .Property(b => b.Id)
+                .IsRequired();
+
+            entity.HasKey(b => b.Id);
+
+            entity.HasOne(b => b.User)
+                .WithOne(b => b.Basket);
+
+            entity.Property(b => b.UserId)
+                .IsRequired(false);
```

Comment:
We agreed that it's not required for basket

Reply:
Removed.